docs/style.css

```diff
@@ -239,78 +297,138 @@ h2:hover a .anchor-hint {
     }
 }
 
-@media (max-width: 720px) {
+@media (max-width: 768px) {
     header {
-        padding: 10px;
+        margin-top: 60px;
+        margin-bottom: 60px;
+    }
+
+    header h1 {
+        font-size: 2rem;
+    }
+
+    header img {
+        height: 1.7rem;
+        margin-left: 0.2rem;
+        margin-right: 0.2rem;
     }
 
     header p {
-        font-size: 0.8rem;
+        font-size: 1rem;
+    }
+
+    footer {
+        margin-top: 60px;
     }
 
     .infi-test {
-        padding: 0.5rem 1rem;
-        margin-top: 0;
+        padding-left: 1rem;
+        padding-right: 1rem;
+        margin-bottom: 60px;
     }
 
     .infi-test ol {
         padding-left: 1.5rem;
     }
 
-    .headline, .details {
-        padding: 0.75rem 0.75rem 0 0.75rem;
-    }
-
     .headline {
         max-width: none;
     }
 
     .block, .infi-test {
         border-width: 2px;
+        margin-left: 0;
+        margin-right: 0;
     }
 
     .block {
         flex-direction: column;
+        margin-top: 132px;
     }
 
     .block-image {
-        display: none;
+        margin: -88px auto 0;
+        width: 176px;
+    }
+
+    .block-content {
+        flex-direction: column;
+        padding: 1rem;
+    }
+
+    .block-anchor-with-offset {
+        top: -88px;
+    }
+
+    .details {
+        margin-left: 0;
     }
 
     .view-overview .blocks {
         grid-template-columns: 1fr;
-        gap: 0;
+        margin-left: 0;
+        margin-right: 0;
     }
 
     .view-overview .blocks .block:last-child {
         grid-column: auto;
     }
 }
 
-.block:nth-child(7n + 1), .infi-test ol .topic-gebruikers:before {
-    background: #D32F46;
+/* Topic coloring */
+
```

Comment:
Nitpick: dat is wat mij betreft de extra bytes en schermruimte niet waard, uit de backets/opmaak/code zelf is dit al wel duidelijk?

```suggestion
```